### PR TITLE
1323 move glad into another repository

### PR DESCRIPTION
**Pull request by kuukitenshi**
# Description

Moved ```Glad``` and ```stb-image``` libs to another repositories, ```cubos-glad``` and ```cubos-stb```, respectively.

## Checklist

- [x] Self-review changes.
- [x] Add entry to the changelog's unreleased section.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,15 +15,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Deadzone for input axis (#844, **@kuukitenshi**).
+- Generic Camera component to hold projection matrix (#1331, **@mkuritsu**).
 - Initial application debugging through Tesseratos (#1303, **@RiscadoA**).
 - Print stacktrace with *cpptrace* on calls to CUBOS_FAIL (#1172, **@RiscadoA**).
 
 ### Fixed
 
 - Spot light angle mismatch between light and shadows (#1310, **@tomas7770**).
 - Spot shadows cause light range cutoff (#1312, **@tomas7770**).
+- Moved Glad and stb-image libs to another repositories, cubos-glad and cubos-stb, respectively (#1323, **@kuukitenshi**).
```
Comment (RiscadoA) [inline CHANGELOG.md:27]: I would not call this a fix - you can add it to a new `### Changed` section, between the Added and Fixed sections :relieved: (we're following this changelog format btw: https://keepachangelog.com/en/1.1.0/)

**Review by RiscadoA — CHANGES_REQUESTED** [review]
Other than the changelog thing, I only have two more things to point out:
- `refactor(build)` would more fitting for the commits instead of `fix(build)` 
- If you can, add the Cubos team as a maintainer (?) to the two repos you created.

Thanks for working on this! python is no more :tada:

**Comment by kuukitenshi** [comment]
> Other than the changelog thing, I only have two more things to point out:
> 
> * `refactor(build)` would more fitting for the commits instead of `fix(build)`
> * If you can, add the Cubos team as a maintainer (?) to the two repos you created.
> 
> Thanks for working on this! python is no more 🎉

I forgot to add the perms mb ^^""

**Comment by RiscadoA** [comment]
@kuukitenshi can you squash the commits into a single one before merging? We try to have a clean git history in main.
Probably the easiest way to get it done is `git reset --soft HEAD~1` and then `git commit --amend` - make sure to update the message of the first commit.

**Review by RiscadoA — APPROVED** [review]
Ty! Looks good to me :rocket:

**Review by GalaxyCrush — APPROVED** [review]

**Review by mkuritsu — APPROVED** [review]
LGTM 👍

**Pull request merged**